backend/Cargo.toml

```diff
@@ -42,6 +42,7 @@ argon2 = "0.5.3"
 password-hash = { version = "0.5.0", features = ["getrandom"] }
 rand = "0.8.5"
 cookie = "0.18.1"
+zip = { version = "2.2.1", default-features = false, features = ["deflate", "time"] }
```

Comment:
Can we use `chrono` instead of `time`?

Reply:
I'm now using chrono, but we no longer get the modified timestamps automatically unfortunately, so I had to add some code for that.